lib/prefill-data.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+'use strict'
+
+/* ::
+import type {
+  PrefillMeta,
+  S3SubmissionData
+} from '../types.js'
+*/
+
+const AWS = require('aws-sdk')
+
+function setPrefillData (
+  prefillMeta /* : PrefillMeta */,
+  data /* : mixed */
+)/* : Promise<mixed> */ {
+  const s3 = new AWS.S3({
+    region: prefillMeta.s3.region,
+    accessKeyId: prefillMeta.credentials.AccessKeyId,
+    secretAccessKey: prefillMeta.credentials.SecretAccessKey,
+    sessionToken: prefillMeta.credentials.SessionToken
+  })
+
+  const objectMeta = {
+    Bucket: prefillMeta.s3.bucket,
+    Key: prefillMeta.s3.key,
+    Body: Buffer.from(JSON.stringify(data), 'UTF-8')
```

Comment:
Should we be setting the following options that are set in the forms renderer when uploading submission data:

- `ContentType: 'application/json'`
- `ServerSideEncryption: 'AES256'`